src/view/login.js

```diff
@@ -0,0 +1,51 @@
+import Layout from "component/layout";
+
+var getUser = function(acct_id, users) {
+    var acctUser = users.filter(function(u) { return u.accountId() == acct_id; });
```

Comment:
I think we should use PSR-2 style for both PHP and JS. Since PSR-2 does not specify how variables are named, I would like to go with :snake: case.